src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+document.addEventListener('click', function (e) {
+  const link = e.target.closest('a');
+
+  if (!link) {
+    return;
+  }
+
+  e.preventDefault();
```

Comment:
Preventing the default action is appropriate here to stop the browser from following the link, but ensure that this behavior is desired for all anchor tags on the page.